src/zql/ivm/graph/operators/full-agg-operators.test.ts

```diff
@@ -0,0 +1,312 @@
+import {expect, test} from 'vitest';
+import {Multiset} from '../../multiset.js';
+import {NoOp} from './operator.js';
+import {DifferenceStreamWriter} from '../difference-stream-writer.js';
+import {
+  FullAvgOperator,
+  FullCountOperator,
+  FullSumOperator,
+} from './full-agg-operators.js';
+
+test('count', () => {
+  const inputWriter = new DifferenceStreamWriter<{x: string}>();
```

Comment:
these commits are prior to https://github.com/rocicorp/rails/pull/55 hence the `writer` and `reader`